backend/src/openarchiefbeheer/destruction/api/serializers.py

```diff
@@ -149,8 +149,10 @@ class Meta:
 
 
 class DestructionListWriteSerializer(serializers.ModelSerializer):
-    reviewer = ReviewerAssigneeSerializer(required=False)
+    add = DestructionListItemWriteSerializer(many=True, required=False)
+    remove = DestructionListItemWriteSerializer(many=True, required=False)
     items = DestructionListItemWriteSerializer(many=True, required=False)
```

Comment:
I feel that if we are going for add/remove, the "items" field should go. Otherwise it is a bit confusing to have them both.

Also, I think that the validation strategy should change a bit:
- For adding we need to check that a case is not in a destruction list including the current one.
- Removing zaken probably should check that there are still more than 0 cases in a list

Reply:
- I didn't check properly yet but I my assumption was, that given the name of the serializer it's shared for all "write" actions, doesn't the create endpoint still required `items`?
- What should happen if we combine "items" and "add"/"remove", should they be able to able to be combined?
- Is it an invalid action to "clear" the destruction list?

| instruction #1 | instruction #2 | result                   |
| ---------------|----------------|----------------|
| `items` |  | `items` 
| `add` |  | existing items + `add` 
| `remove` |  | existing items - `remove` 
| `items` | `add` | `items` + `add`
| `items` | `remove` | `items` - `remove`

Reply:
I was thinking that we could then have a more unified behaviour: `add` always adds to the list items and `remove` always removes (so only the row 2/3 in your table would remain). Then `items` is no longer needed, since you can always obtain the same result with `add`/`remove`. 

I thought that at some point we mentioned that it shouldn't be possible to have empty lists, but I don't think we had a story for it.

Reply:
So:

- Replace `items` behavior/implementation with `add`, removing the ability to replace the entire list.
- Since we can't replace items, lets have the ability to "clear" the list so we can start freshly when editing :) .